op-chain-ops/deployer/integration_test/apply_test.go

```diff
@@ -147,7 +147,7 @@ func TestEndToEndApply(t *testing.T) {
 		{"SuperchainConfigImpl", st.SuperchainDeployment.SuperchainConfigImplAddress},
 		{"ProtocolVersionsProxy", st.SuperchainDeployment.ProtocolVersionsProxyAddress},
 		{"ProtocolVersionsImpl", st.SuperchainDeployment.ProtocolVersionsImplAddress},
-		{"Opsm", st.ImplementationsDeployment.OpsmAddress},
+		{"OpsmProxy", st.ImplementationsDeployment.OpsmProxyAddress},
```

Comment:
This is technically now a bit of a misnomer: `ImplementationsDeployment` because it's a proxy.

Reply:
Yea, we might end up merging `DeploySuperchain` and `DeployImplementations` down the road for this reason, but for now it's ok